src/pytorch/neighbors/getNeighborPairs.py

```diff
@@ -63,18 +71,26 @@ def getNeighborPairs(positions: Tensor, cutoff: float, max_num_neighbors: int =
         If an atom pair is separated by a larger distance than the cutoff,
         the distance is set to `NaN`.
 
+    number_found_pairs: `torch.Tensor`
+        Contains the total  number of pairs found,  which might exceed
+        the  requested  max_num_neighbors,  leaving the  rest  of  the
+        output in an undefined state.
```

Comment:
What about this?: 
        Contains the total number of pairs found in an unspecified order,
        which might exceed the  requested  max_num_neighbors. In this case, the first number_found_pairs
        pairs in the output are valid pairs, but the remaining pairs are omitted.

Reply:
How about this?  "Contains the total number of pairs found.  Be aware that if `check_errors` is False, this might be larger than `max_num_neighbors`.  In that case, the output tensors contain only a subset of the pairs that were found, and the others are omitted.  Which pairs get omitted may vary between invocations."